paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -542,6 +562,73 @@ class ConvMKLDNNTemplateHandler : public MKLDNNHandler {
                                scale_data, mask);
   }
 
+  mkldnn::primitive_attr CreatePostOps(bool fuse_relu,
+                                       bool fuse_residual_conn = false) const {
+    mkldnn::primitive_attr conv_attr;
+    mkldnn::post_ops post_operations;
+    // Fusion with Elementwise layer relies on adding a sum post-operation with
+    // the scale parameter. It is assumed that when fuse_residual_connection is
+    // true, the output tensor contains the data coming from residual
+    // connection. The result of this post_op is:
+    // Output = scale * Output + Conv_Out.
+    if (fuse_residual_conn) {
+      post_operations.append_sum(1.0f);
+    }
+    // Fusion with ReLU layer is executed through the PostOps feature. Create a
+    // PostOps object and configure it to execute an eltwise relu operation.
+    if (fuse_relu) {
+      constexpr float scale = 1.0f;
+      constexpr float negative_slope = 0.0f;
+      constexpr float placeholder = 0.0f;
+      post_operations.append_eltwise(scale, mkldnn::algorithm::eltwise_relu,
+                                     negative_slope, placeholder);
+    }
+    conv_attr.set_post_ops(post_operations);
+    return conv_attr;
+  }
+
+  std::shared_ptr<typename forward_t::primitive_desc>
+  AcquireConvolutionPrimitiveDescriptor(
```

Comment:
Just find that why only `AcquireConvolutionPrimitiveDescriptor` should be place in `mkldnn_reuse.h` while others do not.

Reply:
@tensor-tang AcquireConvolutionPrimitiveDescriptor does use CreatePostOps. CreatePostOps where used in conv_mkldnn as well as conv_transpose_mkldnn . I merged two implementations of CreatePostOps into one so it is shared by conv and conv_transpose  and also because it is used to AcquireConvolutionPrimitiveDescriptor , I decided to put it in mkldnn_reuse.h . Do you suggest other location?

Reply:
That's fine keeping here, if both conv and transpose conv using these functions.